addons/l10n_in_edi_ewaybill/views/account_move_line_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_move_line_tree_l10n_in" model="ir.ui.view">
+        <field name="name">account.move.line.tree.l10n_in</field>
+        <field name="model">account.move.line</field>
+        <field name="arch" type="xml">
+            <tree string="Products" editable="top" create="0">
+                <header>
+                    <button string="Reset to Draft" type="object" name="l10n_in_reset_draft_action"/>
```

Comment:
Why do we need to reset to draft?

Reply:
Whenever a user opens this view, an invoice is always in the 'post' state, but for updating tax from list view invoice state must be 'draft'.